fjord/suggest/providers/trigger/provider.py

```diff
@@ -37,6 +39,20 @@ def build_ga_category(rule):
     return 'trigger_{rule.slug}_{rule.id}'.format(rule=rule)
 
 
+def interpolate_url(url, response):
+    if '{' not in url:
+        return url
```

Comment:
They're two different optimizations, though. The one in `TriggerRedirect.handle_redirect` reduces the work involved--it doesn't pull the response and doesn't try interpolating. This one is here because this function isn't tied to `TriggerRule` and it's nice when functions can stand on their own because they're easier to use elsewhere. In this case we don't use it elsewhere.

Having said that, this should have a docstring.